proxy/cc-proxy.dsc-template

```diff
@@ -1,14 +1,13 @@
 format: 3.0 (quilt)
 Source: cc-proxy
-Version: @VERSION@
+Version: @VERSION@-@REVISION@
 Section: devel
 Priority: optional
 Maintainer: clearlinux.org team <dev@lists.clearlinux.org>
 Standards-Version: 3.9.6
 Build-Depends: dh-make, git, ca-certificates, execstack, devscripts, debhelper, build-essential, dh-autoreconf, make, dh-modaliases
 Homepage: https://clearlinux.org/features/clear-containers
-Debtransform-Tar: cc-proxy-@VERSION_DEB_TRANSFORM@+git.@HASH_TAG@.tar.gz
-DEBTRANSFORM-RELEASE: 1
+Debtransform-Tar: cc-proxy-@DASHLESS_VERSION@.tar.gz
```

Comment:
In this file we use @VERSION@ and @DASHLESS_VERSION@ is this ok? Which is the reason behind this?

Reply:
The debtransform-tar is a directive for indicate the exact name of the source tarball. Since the tarball name has been modified by OBS to comply with spec files naming convention (in this case, without the dash in 3.0.0beta.X), we must use this. The other @VERSION@ is just the normal version of the release (3.0.0-beta.X).